### PR TITLE
Add support for Kobo Libra 2

### PR DESCRIPTION
**Pull request by ubx**
Brief summary of the changes
----------------------------

This PR adds support for the Kobo Libra 2. No Bluetooth support yet.

### CODE REVIEW

**Comment by MaxKellermann** [comment]
A PR branch should not contain merge commit - please clean up the branch.
There are linker failures in the CI.

File: src/Kobo/System.cpp [hunk 1]
```diff
@@ -267,9 +278,11 @@ bool
 KoboWifiOff()
 {
 #ifdef KOBO
+  const char *interface =  (DetectKoboModel() == KoboModel::LIBRA2) ? "wlan0" : "eth0";
```
Comment (MaxKellermann) [inline src/Kobo/System.cpp:281]: There are lots of copies of this one, but we shouldn't have redundant code.

**Comment by ubx** [comment]
Build failure on Linux builds:
```
src/Kobo/Model.hpp:27:2: error: #error This header is only for Kobo builds
```

I'm confused, either we do not include the Kobo Menu in the Linux build, or we allow all necessary files, including Model.hpp.

**Comment by MaxKellermann** [comment]
> Build failure on Linux builds:

7b5fe227e40e1324ca3e86fc71132a9f8cf88d3d turned the linker failure into a compiler failure, so you see the mistake earlier.

> I'm confused, either we do not include the Kobo Menu in the Linux build, or we allow all necessary files, including Model.hpp.

Neither. (Much of) KoboMenu is being developed and debugged on Linux, but obviously the really Kobo-specific things cannot be done, and `Kobo/Model.hpp` must not be used. If your PR adds code which uses more of `Kobo/Model.hpp`, it must be protected with `#ifdef KOBO`.

**Comment by MaxKellermann** [comment]
7bc9dd41ecc61ca16ea7e7c34d2f2666ab895f1c

**Comment by ubx** [comment]
Max, thank you for helping to merge the PR.

**Pull request closed without merging**